### ISSUE TITLE
Bug with gossip sync

### ISSUE THREAD

**Issue by Darth-Coin** [nbd-wtf/obw#34]
### Description
Weird channels names appeared after udpate OBW to v0.2.0-11.

- Normal channel open with a clearnet node [Eternal.Sovereigm](https://amboss.space/node/03bf5f69adc41b4757e2ee88840f167398f4ab3a619fa73b7b25491f6ef349ea16), few weeks ago
- OBW vers. 0.1.8
- Also opened one with Jiraiya and one with SATM
- using clearnet, never Tor

After update OBW to 0.2.0-11 some weird gossip sync happen:
- Jiraiya HC appeared to be etleneum,com (I never open with etleneum.com and is also another IP)
![image](https://user-images.githubusercontent.com/56997124/207979191-df3b9e70-3dc0-4e7e-9f24-9563995e8138.png)

- Eternal.Sovereign node appeared now as [onion] with displaying his old onion address (no longer available)
![image](https://user-images.githubusercontent.com/56997124/207978948-58873c6c-6ac7-4f6e-bd2d-65b7a6d48c6f.png)

Eternal said that only changed the alias few days ago and other wallets / channels are connected well.
Notice in Amboss its old onion address
![image](https://user-images.githubusercontent.com/56997124/207979074-10af510a-5b43-4a29-9f8e-80d11448143c.png)

### Steps I did:
- removed HC etleneum and added again Jiraiya as HC. Worked, all ok, channel online
- Switched to Tor, but the normal channel with Eternal still shows the old address and stays offline.

Will wait few days more to see if the gossip will sync correctly and will report back here.

**Comment by fiatjaf** [comment]
@Darth-Coin if you can do me a favor, please download the apk from https://github.com/nbd-wtf/obw/actions/runs/3810252555 and live with it for a while so we can see if this problem keeps happening. A proper fix would require doing a big refactor, but this should be enough if the problem is just the ACINQ node replaying very old gossip messages on us.

**Comment by Darth-Coin** [comment]
@fiatjaf I install it but is coming as separate new app not overwrite the old one.
The thing is that with the original OBW I can't make export of the channels to import in any other version. See issue #35 
So I can't really test this dev version.
Let me know how should I proceed now. If I can make the backup in OBW I could import into this dev version and see how is doing. But I can't trigger the backup (I have 2 HC + 1 normal chan).
Yes, I could FC that chan (is with Eternalkr, no problem to communicate) but then we can't test to see this fix in v0.2.0 is ok.

**Comment by fiatjaf** [comment]
I'll do that other thing.

**Comment by fiatjaf** [comment]
Actually if you can just use this dev version with a random key and get an HC from Jiraiya and never use it for anything, just wait to see if it changes from Jiraiya to Etleneum eventually.

**Comment by Darth-Coin** [comment]
Ah OK then, I misunderstood. I can do that sure. testing it for a while.

**Comment by Darth-Coin** [comment]
- I updated to new v0.2.1
- made the backup channels
- imported the backup + seed into an other device with v0.2.1
- SATM HC channel OK
- Jiraiya HC channel, imported as etleneum. Removed and add Jiraiya again, worked. OK
- Eternal.Sovereign normal channel is still synced wrong as described in the beginning and is still offline.

**Comment by Darth-Coin** [comment]
> Actually if you can just use this dev version with a random key and get an HC from Jiraiya and never use it for anything, just wait to see if it changes from Jiraiya to Etleneum eventually.

I check almost twice per day using OBW_DEV version, to see if Jiraiya is changing the name. It is not updating.
If you are still looking into fix this issue let me know. I can keep that broken channel open until you find a solution.
Maybe this will help others with the same issue.
But if you think there's no fix for this, also let me know so I can FC the channel and move on.

**Comment by fiatjaf** [comment]
Well, if it is not changing back to Etleneum that is a good thing. Does the channel appear as online? I think you can stop checking now. Thank you very much for your help. Is the channel working in the main OBW too or has that one changed to Etleneum?

**Comment by Darth-Coin** [comment]
yes, the Jiraiya chan is active online and working on both versions (0.2.1 and dev)
Only the normal channel with Eternal node is still offline and showing onion address.

**Comment by Darth-Coin** [comment]
@fiatjaf this morning I tried again to use OBW_DEV version.
As usual, I open OBW, wait to sync the channel (I have only one HC with Jiraiya) and sent the payment.
I get this error, even that I wait several minutes and no sign on top of sync.
![image](https://user-images.githubusercontent.com/56997124/210978973-8a470562-c037-493f-a65b-49ef5fe4766f.png)


Then I tried from another device, the normal OBW (latest version) and payment worked.
